contracts/Aavegotchi/facets/ItemsFacet.sol

```diff
@@ -245,7 +248,7 @@ contract ItemsFacet is Modifiers {
 
             if (existingEquippedWearableId != 0 && s.itemTypes[existingEquippedWearableId].canBeTransferred) {
                 // remove wearable from Aavegotchi and transfer item to owner
-                _removeWearableFromGotchi(_tokenId, existingEquippedWearableId);
+                _removeWearableFromGotchi(_tokenId, existingEquippedWearableId, _sender, _equippedItemsInfo);
```

Comment:
I don't know if we should pass `_sender` as function parameters. If that were the case, we would need it on every single function that required `msg.sender`. What is the Diamond best practice?